src/main/Config.cpp

```diff
@@ -275,6 +275,8 @@ Config::Config() : NODE_SEED(SecretKey::random())
     TESTING_STARTING_EVICTION_SCAN_LEVEL =
         InitialSorobanNetworkConfig::STARTING_EVICTION_SCAN_LEVEL;
 
+    EMIT_SOROBAN_TRANSACTION_META_EXT_V1 = false;
```

Comment:
don't we need to set EMIT_LEDGER_CLOSE_META_EXT_V1 to false as well?

Reply:
Good catch, fixed.